.github/workflows/check-i18n-changes.yml

```diff
@@ -0,0 +1,23 @@
+name: Protect content from manual changes
+
+on: 
+  pull_request:
+    paths:
+    - 'content/**'
+
+jobs:
+  creator-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: 
+        if: github.actor != "nodejs-crowdin" || github.actor != "github-actions"
+        uses: thollander/actions-comment-pull-request@v1
```

Comment:
I think we should close the pull request with the message. Recomend use the [close-pull actions](https://github.com/marketplace/actions/close-pull).

Reply:
We discussed with the team that is not very polite to close PR, and better just to comment it